config/locales/idv/en.yml

```
@@ -82,6 +82,10 @@ en:
         heading: We could not find records matching your personal information.
         warning: Please check the information you entered and try again. Common mistakes
           are an incorrect Social Security number or ZIP Code.
+      setup:
+        fail_html: 'Please <a href="%{contact_form}">fill out our contact form</a> and
```

also in other places, when we embed a link inside i18n, i'd make the entire thing the placeholder, and then build the link using `link_to` in the template"
```suggestion
        fail_html: 'Please %{contact_form_link} and
```
and then like in the erb
```
t('setup.fail_html', contact_form_link(link_to('....'))
```
```